src/components/common/Avatar/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react'
+import { Avatar } from '@material-ui/core'
+
+type BoringAvatarProps = {
+  alt?: string
+}
+const BoringAvatar: React.FC<BoringAvatarProps> = ({ alt }) => {
+  const [colors] = useState<string[]>(
+    Array.from({ length: 6 }, () => Math.floor(Math.random() * 16777215).toString(16)),
```

Comment:
Why this random logic? 

also, nit, since you are creating a component call `BoringAvatar`, should you call the file with the same name?